test/PidTest.cpp

```diff
@@ -0,0 +1,116 @@
+/**
+ *  @file PidTest.cpp
+ *  @brief PidGainクラス,Pidクラスをテストする
+ *  @author Hisataka-Hagiyama,uchyam
+ */
+
+#include "Pid.h"
+#include <gtest/gtest.h>
+
+namespace etrobocon2021_test {
+  TEST(PidGainTest, gain)
+  {
+    PidGain myPidGain(0.1, 0.03, 0.612);
+    EXPECT_DOUBLE_EQ(0.1, myPidGain.kp);
+    EXPECT_DOUBLE_EQ(0.03, myPidGain.ki);
+    EXPECT_DOUBLE_EQ(0.612, myPidGain.kd);

```

Comment:
```suggestion
    double expected_p = 0.1;
    double expected_i = 0.03;
    double expected_d = 0.612;
    PidGain actualPidGain(expected_p, expected_i, expected_d);
    EXPECT_DOUBLE_EQ(expected_p, actualPidGain.kp);
    EXPECT_DOUBLE_EQ(expected_i, actualPidGain.ki);
    EXPECT_DOUBLE_EQ(expected_d, actualPidGain.kd);

```
テストコードにおいてマジックナンバーは避けるべきなので，ちょっと面倒かもしれないけど変数に入れてからテストしてください
下の2つとかも